content/en/docs/guides/complex-component-structure-deployment.md

```diff
@@ -0,0 +1,839 @@
+    resourceRef:
+      name: config
+      referencePath:
+      - name: backend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: manifests
+      referencePath:
+      - name: backend
+      version: 1.0.0
+```
+
+```yaml
+# Configuration
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Configuration
+metadata:
+  name: backend-configuration
+  namespace: ocm-system
+spec:
+  configRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: config
+      referencePath:
+      - name: backend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Localization
+    name: backend-localization
+    namespace: ocm-system
+```
+
+```yaml
+# FluxDeployment
+apiVersion: delivery.ocm.software/v1alpha1
+kind: FluxDeployer
+metadata:
+  name: backend-kustomization
+  namespace: ocm-system
+spec:
+  kustomizationTemplate:
+    prune: true
+    targetNamespace: default
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Configuration
+    name: backend-configuration
+    namespace: ocm-system
+```
+
+And that's it.
+
+### Frontend
+
+Same for the Frontend
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Localization
+metadata:
+  name: frontend-localization
+  namespace: ocm-system
+spec:
+  configRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: config
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: manifests
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+```
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: Configuration
+metadata:
+  name: frontend-configuration
+  namespace: ocm-system
+spec:
+  configRef:
+    kind: ComponentVersion
+    name: podinfocomponent-version
+    namespace: ocm-system
+    resourceRef:
+      name: config
+      referencePath:
+      - name: frontend
+      version: 1.0.0
+  interval: 10m0s
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Localization
+    name: frontend-localization
+    namespace: ocm-system
+```
+
+```yaml
+apiVersion: delivery.ocm.software/v1alpha1
+kind: FluxDeployer
+metadata:
+  name: frontend-kustomization
+  namespace: ocm-system
+spec:
+  kustomizationTemplate:
+    prune: true
+    targetNamespace: default
+  sourceRef:
+    apiVersion: delivery.ocm.software/v1alpha1
+    kind: Configuration
+    name: frontend-configuration
+    namespace: ocm-system
+```
+
+### Redis
+
+Redis is exactly the same as the above two. Just with different names and pointing to the redis resource. Try creating
```

Comment:
for a guide we should provide those, maybe just the parts that are different.

Reply:
we are providing those: https://github.com/open-component-model/podinfo/

Reply:
Actually, I'll add all the components to the podinfo app.